pkg/wing/server/server.go

```diff
@@ -131,11 +110,5 @@ func (o WingServerOptions) RunWingServer(stopCh <-chan struct{}) error {
 		return err
 	}
 
-	/*server.GenericAPIServer.AddPostStartHook("start-sample-server-informers", func(context genericapiserver.PostStartHookContext) error {
-		config.GenericConfig.SharedInformerFactory.Start(context.StopCh)
-		return nil
-	})
-	*/
```

Comment:
Didn't realise this was commented out, and feel like it shouldn't be commented out... but that's something for another PR. Can you just leave this comment block as-is in this PR? No reason to delete here.

Reply:
When I comment it out there is null pointer expection, never bother as things just work for me